acados/sim/sim_irk_integrator.c

```diff
@@ -567,20 +567,14 @@ int sim_irk(void *config_, sim_in *in, sim_out *out, void *opts_, void *mem_, vo
                 impl_ode_res_out.xi = ii*nx; // store output in this posistion of rG
                 // compute the residual of implicit ode at time t_ii
                 if ( !((opts->jac_reuse & (ss==0) & (iter==0)) | (!opts->jac_reuse)) )
-                { // otherwise eval the ode together with the jacobians within next if
+                { 
                     acados_tic(&timer_ad);
                     model->impl_ode_fun->evaluate(model->impl_ode_fun, impl_ode_type_in, impl_ode_in, impl_ode_fun_type_out, impl_ode_fun_out);
                     timing_ad += acados_toc(&timer_ad);
-                    // fill in elements of rG
                 }
-
-                if ( (opts->jac_reuse & (ss==0) & (iter==0)) | (!opts->jac_reuse) )
-				{
-                    // compute the jacobian of implicit ode
```

Comment:
what are these changes about?

Reply:
i made this change to increase the readability of the code.
instead of having 2 complementary ifs; if+else is much more clear..